documentation/modules/con-securing-kafka-authentication.adoc

```diff
@@ -99,16 +99,19 @@ When a `KafkaUser.spec.authentication.type` is configured with `scram-sha-512` t
 [id='assembly-kafka-broker-listener-network-policies-{context}']
 == Network policies
 
-Strimzi automatically creates a `NetworkPolicy` resource for every listener that is enabled on a Kafka broker.
-By default, a `NetworkPolicy` grants access to a listener to all applications and namespaces.
+By default, Strimzi automatically creates a `NetworkPolicy` resource for every listener that is enabled on a Kafka broker.
 
-If you want to restrict access to a listener at the network level to only selected applications or namespaces,
+A `NetworkPolicy` grants a listener access to applications in all namespaces.
```

Comment:
Maybe one more comment to this ... Do you think that it will be clear that it is the applications which can connect to the listener and not the other way around? I think if you know what listener is the direction should be clear. But I'm not sure how many readers will get that on the first look.

Reply:
I've tried rephrasing as follows:

> By default, Strimzi automatically creates a `NetworkPolicy` resource for every listener that is enabled on a Kafka broker.
> This `NetworkPolicy` allows applications to connect to listeners in all namespaces. 

Does that make sense?

Reply:
That sounds great, thanks.